_backend/email/os-payment.php

```diff
@@ -81,11 +83,15 @@ function email_os_payment (\Stripe\PaymentIntent $intent) {
         'merge_language' => 'handlebars'
     );
 
-    $res = $mandrill->messages->sendTemplate('os-purchase', $req, $message);
+    $res = $mailchimp->messages->sendTemplate([
+        "template_name" => "os-purchase",
+        "template_content" => $req,
+        "message" => $message
+    ]);
 
     foreach ($res as $mail) {
-        if (isset($mail['reject_reason']) && $mail['reject_reason'] !== '') {
-            throw new Exception($mail['reject_reason']);
+        if (isset($mail->reject_reason) && $mail->reject_reason !== '') {
```

Comment:
I don't really know PHP outside of trying to use it on this repo but is there a reason for changing the notation here? I understand that they are both forms of object notation.

Reply:
PHP has a history of objects and arrays (they are kind of the same thing in the language), but typically you would use `$object->function()` and `$object->key` but `$array['key']`. This is probably just to be consistent so you know what you're working with.